content/protobuf/language-mapping/rpc-method.md

```diff
@@ -0,0 +1,70 @@
+---
+title: RPC method
+description: Learn how RPC methods are mapped to C#.
+---
+
+## C# mapping
+
+A Protobuf RPC method named _OpName_ in service `Greeter` is mapped to abstract method *OpName*Async in the interface
+`IGreeter` and to abstract method *OpName*Async in interface `IGreeterService`.
```

Comment:
```suggestion
A Protobuf RPC method named _OpName_ in service `Greeter` is mapped to abstract method *OpName*Async in an interface named
`IGreeter` and to abstract method *OpName*Async in an interface named `IGreeterService`.
```

Reply:
Updated to use RpcName. I don't like "an interface named" here.